### DISCUSSION TITLE
How can I show all calendar border and set grid height in theme?

### DISCUSSION THREAD

**Discussion by st474ddr** [nhn/tui.calendar#1294] [Q&A]
## Version
@toast-ui/vue-calendar: ^1.1.1 

## Test Environment
edge, mac OS Monterey 12.5

## Current Behavior
`
// template
<Calendar
          ref="calendar"
          v-loading="loading"
          :view="viewMode"
          :month="monthTitle"
          :schedules="events"
          :theme="theme"
          :use-creation-popup="false"
          :use-detail-popup="false"
          @clickSchedule="onClickSchedule"
          @beforeCreateSchedule="onBeforeCreateSchedule"
/>

<script>
theme ={ 
  'common.border': '1px solid',
  'month.schedule.height': '60px',
}
</script>
`

The border only show inside the calendar. I want full calendar border
And the schedule grid doesn't change height.

## Expected Behavior
border 
![image](https://user-images.githubusercontent.com/17706261/191031244-bdfb7fdc-d497-4c86-b4aa-8d1cfbf0b736.png)

schedule grid height
![image](https://user-images.githubusercontent.com/17706261/191031847-764e9d6e-577a-48ba-b969-3b4303f928f1.png)

**Comment by adhrinae** [comment]
@st474ddr 

It's working for me 🤷‍♂️ 

![image](https://user-images.githubusercontent.com/14539203/191137977-476e4087-8cea-41d0-8631-6d757b1b6123.png)

Make sure there's no other element hiding the border. Or you can just set the container's style directly.

![image](https://user-images.githubusercontent.com/14539203/191138475-6fb82d7b-ee6f-49d5-8db4-295a3ee221bd.png)

And There's no option to set the height of the row height.

Reference: https://github.com/nhn/tui.calendar/issues/906#issuecomment-933946119
